src/applications/personalization/profile/components/personal-information/email-addresses/EmailInformationSection.jsx

```diff
@@ -3,11 +3,11 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import React from 'react';
 
-import { FIELD_NAMES, FIELD_TITLES } from '@@vap-svc/constants';
+import { FIELD_NAMES } from '@@vap-svc/constants';
+import ContactInformationField from '~/applications/personalization/profile/components/personal-information/ContactInformationField';
```

Comment:
Again maybe a relative import would be cleaner here? But that's a matter of opinion.